brutils/ibge/uf.py

```diff
@@ -1,4 +1,6 @@
-from brutils.data.enums.uf import CODE_TO_UF
+from unidecode import unidecode
+
+from brutils.data.enums.uf import CODE_TO_UF, UF
 
 
 def convert_code_to_uf(code):  # type: (str) -> str | None
```

Comment:
Apesar do brutils não utilizar algo como o ```mypy``` para verificação de tipos de maneira estática,  recomendo fortemente utilizar tipagem no código mesmo ao invés de docstrings. 
```suggestion
def convert_code_to_uf(code: str)  -> str | None:
```